packages/inertia-react/index.d.ts

```diff
@@ -86,3 +86,29 @@ export interface InertiaFormProps<TForm = Record<string, any>> {
 }
 
 export function useForm<TForm = Record<string, any>>(initialValues: TForm): InertiaFormProps<TForm>;
+
+export interface AppSetupPageProps {
+    initialPage: Inertia.Page<Inertia.PageProps>,
```

Comment:
`Inertia.PageProps` is unnecessary here, as now it's being double defined and merged (directly overriding the defaults). This option is really only ever necessary in userland to include shared props.

```suggestion
    initialPage: Inertia.Page,
```